src/templates/kit/screens/TokensScreen.js

```diff
@@ -218,45 +245,45 @@ function Tokens({
             )}
           </Field>
         </div>
-        <Field
-          label={
-            <div
+      </div>
+      <Field
+        label={
+          <div
+            css={`
+              width: 100%;
+              ${fieldsLayout}
```

Comment:
Should we always add `;` to the end of computed styles? I am thinking on an error with code formatting that happened because it was not the last prop being set and `prettier` thought it meant something else.

Reply:
Agreed, it could be a good idea to start doing that since styled-components is happy with it, and it is a valid CSS syntax :+1: